### PR TITLE
Updated the README.md file to describe my repo.

### PR DESCRIPTION
**Pull request by saka01**
image
Perfect working simplest page

### CODE REVIEW

**Comment by saka01** [comment]
@FluffySheep

File: portfolio/.gcloudignore [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+#
+.gcloudignore
+# If you would like to upload your .git directory, .gitignore file or files
+# from your .gitignore file, remove the corresponding line
+# below:
+.git
+.gitignore
+
+# Target directory for maven builds
+target/
```
Comment (FluffySheep) [inline portfolio/.gcloudignore]: minor: add a new line at the end of the file, this is common practice for files uploaded to git. It doesn't do anything except make sure that github will not complain about styling here.

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -8,11 +8,12 @@
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
+      <h1 class="name">I AM MOSHOOD SAKA.</h1> 
+      <img class="soon" src = "/images/soon.png"/>
```
Comment (FluffySheep) [inline portfolio/src/main/webapp/index.html]: "soon" is a fairly generic tag. Consider making this more specific about what it is being used for on the page similar to how you see ids such as "greeting-container" below.

File: portfolio/src/main/webapp/style.css [hunk 3]
```diff
@@ -1,9 +1,34 @@
...
+    text-align: center;
+    width: 450px;
+    color: white;
+    /* border-spacing: 1em .5em; */
+    padding: 30px;
+    border: 5px solid white; 
+    transform: translate(-50%, -50%);
+
+}
+.soon{
```
Comment (FluffySheep) [inline portfolio/src/main/webapp/style.css]: similar with this naming.

File: portfolio/src/main/webapp/style.css [hunk 4]
```diff
@@ -1,9 +1,34 @@
...
+    height: 500px;
+    width: 500px;
+    position: absolute;
+    top: 62.5%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+}
+
+/* #content {
```
Comment (FluffySheep) [inline portfolio/src/main/webapp/style.css]: if you aren't using code, please remove it instead of commenting it out. thanks!

**Review by FluffySheep — COMMENTED** [review]
It looks like you make some changes in the UI, please add a screenshot to your pull request description. Thanks!

File: portfolio/src/main/webapp/style.css (hunk 4 above)
Comment (saka01) [reply portfolio/src/main/webapp/style.css]: taken out the comments from styles.css

File: portfolio/src/main/webapp/style.css (hunk 3 above)
Comment (saka01) [reply portfolio/src/main/webapp/style.css]: changed the naming to "web-in-progress"

File: portfolio/src/main/webapp/index.html (hunk 2 above)
Comment (saka01) [reply portfolio/src/main/webapp/index.html]: changed the naming to "web-in-progress"

File: portfolio/.gcloudignore (hunk 1 above)
Comment (saka01) [reply portfolio/.gcloudignore]: Added extra line

**Review by FluffySheep — APPROVED** [review]

**Pull request merged**